droidlet/lowlevel/hello_robot/hello_robot_mover.py

```diff
@@ -14,18 +14,17 @@
 
 import cv2
 from droidlet.shared_data_structs import ErrorWithResponse
-from agents.argument_parser import ArgumentParser
+#from agents.argument_parser import ArgumentParser
```

Comment:
this is a modified version of `argparse.ArgumentParser`. Use this instead?

Reply:
don't need the full agent argument parser here, its just for testing the mover interactively from ipython